File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    
    for (int i = 1; i <= h; i++)
    {
        //print the space 
        for (int j = 1; j <= h - i ; j++)
        {
            printf(" ");
        }
        //print the hash
        for (int k = 1; k <= i ; k++)
        {
            printf("#");
        }
        //print the gap
        for (int m = 0; m < 1 ; m++)
        {
            printf("  ");
        }
        
        //print the hash
        for (int n = 1; n <= i ; n++)
        {
            printf("#");
        }
        printf("\n");

    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# Get integer of height from user input
while True:
    h = get_int("Height: ")

    if h in range(1, 9):
        break

# print the pyramid in concatenate the string

for i in range(1, h + 1):
    print(" " * (h - i) + "#" * i + "  " + "#" * i)

'''
# print the pyramid
for i in range(1, h + 1):
    # print space
    for j in range(h - i):
        print(" ", end="")
    # print hash
    for k in range(i):
        print("#", end="")
    # print gap
    for m in range(1):
        print("  ", end="")
    # print hash
    for n in range(i):
        print("#", end="")
    # print endline char
    print()
'''